locales/en/yourloops.json

```diff
@@ -51,6 +51,7 @@
   "aria-team-members-dialog-rmmember-title": "Remove a healthcare professional from the team {{teamName}}",
   "aria-team-members-dialog-switch-role-title": "Remove myself as {{teamName}} administrator",
   "aria-toggle-password-visibility": "Toggle password visibility",
+  "birthdate": "Date of birth",
```

Comment:
opt: the key `patient-profile-birthdate` has the same value and thereby could be replaced by `birthdate`